src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
How does this static::getInstance() deal with scenarios where a static call is made for a different model class? For example I make a call for Customer::find() and then follow that up with a call to StockItem::find(). For the second call static::$instance is set to an instance of Customer but I am now looking to find a StockItem. 

I may be missing something here but this was the issue that lead me down the track of effectively dumping static::getInstance.